girderformindlogger/web_client/src/templates/body/userAccount.pug

```diff
@@ -25,8 +25,8 @@ ul.g-account-tabs.nav.nav-tabs
     .g-user-info-container.g-user-settings-container
       form#g-user-info-form(role="form")
         .form-group
-          label(for="g-login") Login
-          p.form-control-static= user.get('login')
+          label(for="g-login") User Name
+          input#g-login.input-sm.form-control(type="text", value=user.get('login'))
```

Comment:
This is good but unnecessary. This interface is vestigial from the upstream Girder project and not one in use for MindLogger.